website/README.md

```diff
@@ -18,14 +18,26 @@ The Beam website is built using [Jekyll](http://jekyllrb.com/). Additionally,
 for additional formatting capabilities, this website uses
 [Twitter Bootstrap](http://getbootstrap.com/).
 
-### Repository Structure
+Documentation generated from source code, such as Javadoc and Pydoc, is stored
+separately on the [beam-site
+repository](https://github.com/apache/beam-site/tree/asf-site/content/documentation/sdks).
```

Comment:
Eventually this will be migrated to a different path on a separate branch. The current plan is https://github.com/apache/beam-site/tree/release-docs.

The reason I'm creating a new branch rather than re-use the existing `asf-site` branch is to ease the migration; currently we're publishing the contents of `asf-site`, so we're not able to easily restructure those contents without breaking the live site.